Source/Core/Core/PowerPC/GDBStub.h

```diff
@@ -8,11 +8,9 @@
 
 #include "Common/CommonTypes.h"
 
-#ifdef _WIN32
-#define SIGTRAP 5
-#define SIGTERM 15
-#define MSG_WAITALL 8
-#endif
+#define GDB_SIGTRAP     ( 5)
+#define GDB_SIGTERM     (15)
+#define GDB_MSG_WAITALL ( 8)
```

Comment:
Thoughts on making those either `constexpr` or `enum` instead of defines/macros?

Reply:
I simply used macros because the original ones were, too. I guess I could change it, if you prefer that.